taxi/urls.py

```diff
@@ -46,6 +50,23 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("drivers/create/", DriverCreateView.as_view(), name="driver-create"),
+    path(
+        "drivers/<int:pk>/delete/",
+        DriverDeleteView.as_view(),
+        name="driver-delete"
+    ),
+    path(
+        "drivers/<int:pk>/license-update/",
```

Comment:
I guess it should be just `update`. You may think about this as if only update license feature is implementing right now, and later other update params will be available

Reply:
But according to the 1st point of the task we should implement Create and Delete views for Driver